move_tree.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'square'
+
+class MoveTree
+
+  attr_reader :root, :max_depth, :num_of_nodes
+
+  def initialize(coordinates,max_depth)
+    @root = Square.new(coordinates[0],coordinates[1],0,nil,[])
```

Comment:
Looks like we were thinking about the same thing in this section. I may need to establish a hard and fast constant root though.